src/main/java/com/google/maps/model/AddressComponent.java

```diff
@@ -23,7 +24,10 @@
  * href="https://developers.google.com/maps/documentation/geocoding/intro">Google Maps Geocoding API
  * Developer's Guide</a> for more detail.
  */
-public class AddressComponent {
+public class AddressComponent implements Serializable {
+  /** The default serialVersionUID */
+  private static final long serialVersionUID = 1L;
```

Comment:
Hey @markmcd and @samthor, is this a legit way of introducing `serialVersionUID` for a class that is just now implementing `Serializable`? I believe this shouldn't break anyone, but I'd like confirmation... =)

Reply:
I think this is fine. It only mattered if `AddressComponent` was previously `Serializable`, which was not.

FWIW, my limited reading suggests that the reason most `serialVersionUID` fields are random super high numbers is that IDEs will create them using the previous default Java hashing implementation. But again, as the class was never serializable, anything is fine here.

Reply:
Thanks Sam, I concur.